assets/component-cart.css

```diff
@@ -87,19 +87,26 @@ cart-items {
 
 .cart__note {
   display: block;
+  height: 100%;
 }
 
 .cart__note label {
   display: flex;
   align-items: flex-end;
+  position: relative;
   line-height: 1;
   height: 1.8rem;
-  margin-bottom: 2rem;
+  bottom: 2rem;
   color: rgba(var(--color-foreground), 0.75);
 }
 
+.cart__note:after {
+  border-bottom-right-radius: 0;
+}
+
 .cart__note .field__input {
-  padding: 1rem;
+  padding: calc((var(--inputs-border-width)) + 0.5rem);
+  height: 100%;
 }
```

Comment:
The border now seem to cover the bottom right icon. ![](https://screenshot.click/22-01-urqpn-xoeus.png)